### PR TITLE
Downgrade RKE2 to v1.21.11+rke2r1

### PR DESCRIPTION
**Pull request by bk201**
Depends on https://github.com/harvester/os2/pull/35 (Note I include a temporary image in commits)

**Related issue**: https://github.com/harvester/harvester/issues/2112

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM and can we update the base OS image, thanks. 
Also loop @TachunLin  @lanfon72 because of the [ETCD data inconsistency issue](https://groups.google.com/a/kubernetes.io/g/dev/c/B7gJs88XtQc/m/rSgNOzV2BwAJ?pli=1) we need to downgrade the RKE2 version to v1.21.11 which uses the ETCD 3.4.x, which means we will need to run the upgrade regression test probably with the `v1.0-head.ISO` image later, thanks.

File: scripts/package-harvester-os [hunk 1]
```diff
@@ -14,7 +14,7 @@ source ${SCRIPTS_DIR}/version-rancher
 source ${SCRIPTS_DIR}/version-harvester ${TOP_DIR}/../harvester
 source ${SCRIPTS_DIR}/version-monitoring
 
-BASE_OS_IMAGE="rancher/harvester-os:20220406"
+BASE_OS_IMAGE="bk201z/harvester-os:20220406-60b74cd"
```
Comment (weihanglo) [inline scripts/package-harvester-os:17]: Build succeeded and we can now replace it.

https://drone-publish.rancher.io/harvester/os2/46/1/2


```suggestion
BASE_OS_IMAGE="bk201z/harvester-os:20220407-ae38642"
```

**Review by weihanglo — APPROVED** [review]
LGTM

File: scripts/package-harvester-os (hunk 1 above)
Comment (weihanglo) [reply scripts/package-harvester-os:17]: Need a tag on harvester/os2 repo

Comment (bk201) [reply scripts/package-harvester-os]: Sure, The image is published and used.

**Pull request merged**